src/WebCrawler.Shared.DevOps/CrawlerBootstrapper.cs

```diff
@@ -19,33 +25,89 @@ namespace WebCrawler.Shared.DevOps
     /// </summary>
     public static class CrawlerBootstrapper
     {
-        public static Akka.Configuration.Config ApplyOpsConfig(this Akka.Configuration.Config previousConfig)
+        public static AkkaConfigurationBuilder WithOps(this AkkaConfigurationBuilder builder, ClusterOptions clusterOptions, IConfiguration config)
         {
-            var nextConfig = previousConfig.BootstrapFromDocker();
-            return OpsConfig.GetOpsConfig().ApplyPhobosConfig().WithFallback(nextConfig);
+            // Akka.Cluster split-brain resolver configurations
+            clusterOptions.SplitBrainResolver = new KeepMajorityOption();
+            
+            builder
+                .AddHoconFile("shared.hocon", HoconAddMode.Prepend)
+                .AddHocon(config.GetSection("Akka"), HoconAddMode.Prepend)
+                .WithClustering(clusterOptions)
+                .BootstrapFromDocker(config)
+                .AddPetabridgeCmd(pbm =>
+                {
+                    // enable cluster management commands
```

Comment:
Guess we need to add something to Petabridge.Cmd.Host here to make it possible to set the listening address programmatically